botocore/data/rds/2014-10-31/waiters-2.json

```diff
@@ -98,20 +86,87 @@
         }
       ]
     },
-    "DBSnapshotCompleted": {
```

Comment:
This looks like a breaking change as we are renaming the waiter from `DBSnapshotCompleted` to `DBSnapshotAvaliable`. However, I am not sure how useful it was given its configuration was this:
```
    "DBSnapshotCompleted": {
      "delay": 15,
      "operation": "DescribeDBSnapshots",
      "maxAttempts": 40,
      "acceptors": [
        {
          "expected": "DBSnapshotNotFound",
          "matcher": "error",
          "state": "success"
        },
        {
          "expected": "available",
          "matcher": "pathAll",
          "state": "success",
          "argument": "DBSnapshots[].Status"
        }
      ]
}
```
Looks like the first acceptor should have been a failure?

Reply:
Ah, I missed that one.  This is one botocore had that wasn't upstream.  I'm going to just add it back verbatim.